cms/wp/db/backup/download.php

```diff
@@ -0,0 +1,13 @@
+<?php
+/**
+ * Download the latest database backup file.
+ *
+ * @package wordpress
+ */
+
+namespace Deployer;
+
+task('wp:db:backup:download', function () {
```

Comment:
We have several db-related tasks that are both generic (not WP-specific) and are closely related to each other.  Might I recommend placing them all into the same file to simplify maintenance?

Reply:
The issue here is the executor is different.  For drupal it's drush, and for wordpress it's wp.  So even though they are close , they are different.

Reply:
Good point.

That executor is only used in the tasks that create and restore the backups.  The other tasks which download or rotate dumps don't use it.  So perhaps a better suggestion might be combining those two sets of tasks into two separate files:

- A generic one that handles downloading and rotating
- A CMS-specific one for WP

WDYT?